ggml/src/ggml-sycl.cpp

```diff
@@ -3658,6 +3658,10 @@ static void ggml_sycl_mul_mat(ggml_backend_sycl_context & ctx, const ggml_tensor
     use_mul_mat_q = use_mul_mat_q && (src1->ne[1] <= MMQ_MAX_BATCH_SIZE);
 #endif // SYCL_USE_XMX
 
+    // mmvq path is faster in the Nvidia backend but slower on the Intel backend
+    if (ctx.stream()->get_backend() == sycl::backend::ext_oneapi_cuda)
```

Comment:
It's working solution.

But I think if we need an interface to return the hardware info: return hardware vendor (Intel, NV, AMD) or family (Arc, Max series).
Like use  ggml_sycl_device_info() handle it.
Maybe it could be refactor in another PR.